src/lib/core/scroll/scrollable.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Directive, ElementRef, OnInit, OnDestroy, ModuleWithProviders,
+  NgModule
+} from '@angular/core';
+import {Subject} from 'rxjs/Subject';
+import {Observable} from 'rxjs/Observable';
+import {Scroll} from './scroll';
+
+
+/**
+ * Sends an event when the directive's element is scrolled. Registers itself with the Scroll
+ * service to include itself as part of its collection of scrolling events that it can be listened
+ * to through the service.
+ */
+@Directive({
+  selector: '[md-scrollable]'
+})
+export class Scrollable implements OnInit, OnDestroy {
+  /** Subject for notifying that the element has been scrolled. */
+  private _elementScrolled: Subject<Event> = new Subject();
+
+  constructor(private _elementRef: ElementRef, private _scroll: Scroll) {}
+
+  ngOnInit() {
+    this._scroll.register(this);
+    this._elementRef.nativeElement.addEventListener('scroll', (e: Event) => {
```

Comment:
Instead of binding the event directly, [I think you could use `Observable.fromEvent`](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/fromevent.md). Also I think the same can be done above in the service.